README.md

```diff
@@ -761,22 +756,22 @@ export OPENAI_API_ORG="org-..."
 
 #### spacy.REST.v1
 
-This default backend uses `requests` and a simple retry mechanism to access an API.
+This default model uses `requests` and a simple retry mechanism to access an API.
 
 ```ini
-[components.llm.backend]
-@llm_backends = "spacy.REST.v1"
-api = "OpenAI"
-config = {"model": "gpt-3.5-turbo", "temperature": 0.3}
+[components.llm.model]
+@llm_models = "spacy.gpt-3-5.v1"
+name = "gpt-3.5-turbo"
+config = {"temperature": 0.3}
 ```
 
-| Argument    | Type             | Default | Description                                                                                                          |
-| ----------- | ---------------- | ------- | -------------------------------------------------------------------------------------------------------------------- |
-| `api`       | `str`            |         | The name of a supported API. In v.0.1.0, only "OpenAI" is supported.                                                 |
-| `config`    | `Dict[Any, Any]` | `{}`    | Further configuration passed on to the backend.                                                                      |
-| `strict`    | `bool`           | `True`  | If `True`, raises an error if the LLM API returns a malformed response. Otherwise, return the error responses as is. |
-| `max_tries` | `int`            | `3`     | Max. number of tries for API request.                                                                                |
-| `timeout`   | `int`            | `30`    | Timeout for API request in seconds.                                                                                  |
+| Argument    | Type             | Default | Description                                                                                                                                               |
+| ----------- | ---------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`      | `str`            |         | Model name. This might be any of the supported variants for this particular model (e. g. "gpt-3-5-turbo" or "gpt-3.5-turbo-16k" for ""spacy.gpt-3-5.v1"). |
```

Comment:
```suggestion
| `name`      | `str`            |         | Model name. This might be any of the supported variants for this particular model (e. g. "gpt-3-5-turbo" or "gpt-3.5-turbo-16k" for "spacy.gpt-3-5.v1"). |
```